cynthion/python/src/gateware/analyzer/top.py

```diff
@@ -328,8 +328,11 @@ def elaborate(self, platform):
         # Create a USB analyzer.
         m.submodules.analyzer = analyzer = USBAnalyzer(utmi_interface=utmi)
 
-        # Add a special stream clock converter for 'sync' to 'usb' crossing.
+        # Follow this with a HyperRAM FIFO for additional buffering.
         reset_on_start = ResetInserter(analyzer.discarding)
+        m.submodules.psram_fifo = psram_fifo = reset_on_start(HyperRAMPacketFIFO())
```

Comment:
We should set `out_fifo_depth` to some big enough value to contain bursts. If we don't, all read bursts will be 2 words at most.

Reply:
I set it to 4096, which is more than needed - but I think it makes sense to have a bit of capacity there, to make sure the HyperRAM still gets to do large bursts on the read side even if the host's endpoint reading is a bit jittery.